about.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone Project</title>
+    <link rel="stylesheet" href="abc.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    
+</head>
+<body>
+
+<section id="about">
+    <nav class='nav-border-container'>
+        <div class="nav-logos"><img src="images/facebook (1).png" alt="facebook logo" class="img-color"><img src="images/twitter (1).png" alt="twitter logo" class="img-color"><h2 class="nav-head">English</h2><h3 class='nav-head'>My Page</h3><h4 class='nav-head' id="logout">Logout</h4></div>
+    </nav>
+    <div class="desktop-navbar-container"><a href="index.html"><img src="images/atom.png" alt="atom logo" ></a><nav class="desktop-nav-head-container"><a class="desktop-nav-head" href="index.html">Home</a><h3 class="desktop-nav-head">Program</h3><h4 class="desktop-nav-head">Join</h4><h5 class="desktop-nav-head">Sponsor</h5><h6 class="desktop-nav-head">News</h6><h2 class="desktop-nav-head">ti campaign</h2></nav></div>
+    <nav class="nav-bar-about">
+        <div><span class="material-symbols-outlined" id="menu-bar2" style="font-size: 48px;color:#272A31;">menu</span></div>
+    </nav>
+    <div id='about-top-container'>
+    <h2 class="top-heading2">The ti 2022 the greatest of all.</h2>
+    <p class="about-paragraph">The major tournament of dota 2 brings all the major and minor teams under one roof and offers the chance to witness the greatest of all multiplayers clashes including some previous rivalaries and top of the teams from all corners of the world gather and compete to win the ageis and this November finally the wait would be over and make yourself ready for witnessing this mega event</p>
+<p class="about-paragraph-2">Email us for futher information about the tournament we would love to have feedback.
+</p>
+<a class="about-paragraph-3">@dota-2.org.</a>
```

Comment:
[Optional] Please use an indentation of two spaces in all your documents. This is a convention and also [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).